web/hand_tool.js

```diff
@@ -35,30 +35,47 @@ var GrabToPan = grabToPan.GrabToPan;
 var Preferences = preferences.Preferences;
 var SecondaryToolbar = secondaryToolbar.SecondaryToolbar;
 
-var HandTool = {
-  initialize: function handToolInitialize(options) {
-    var toggleHandTool = options.toggleHandTool;
+/**
+ * @typedef {Object} HandToolOptions
+ * @property {HTMLDivElement} container - The document container.
+ * @property {HTMLButtonElement} toggleHandTool - The button element for
+ *                                                toggling the hand tool.
+ */
+
+/**
+ * @class
+ */
+var HandTool = (function HandToolClosure() {
+  /**
+   * @constructs HandTool
+   * @param {HandToolOptions} options
+   */
+  function HandTool(options) {
+    this.container = options.container;
+    this.toggleHandTool = options.toggleHandTool;
+
```

Comment:
Done in the new commit. It was also a bit weird that the value of the member variable was either `true` or `null`, so I changed that to `true` and `false`.